sites/public/.env.template

```diff
@@ -6,7 +6,7 @@ NEXTJS_PORT=3000
 MAPBOX_TOKEN=
 LANGUAGES=en,es,zh,vi
 IDLE_TIMEOUT=5
-COUNTY_CODE=Alameda
+JURISDICTION_NAME=Alameda
```

Comment:
I wonder if our template should be something like 'Development' rather than any reference to Alameda?